src/screens/Contact/Contact.js

```diff
@@ -441,6 +432,13 @@ class Contact extends React.Component<Props, State> {
                     icon={iconSend}
                     onPress={() => this.onSendPress(displayContact)}
                   />
+                  <CircleButton
+                    disabled={disableSend}
```

Comment:
Shouldn't it be `disabled={!displayContact.status}` ?

Reply:
😊 yeah - updated